assets/stories/skate-components/newRouteLadderHeader.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { NewHeader as RouteLadderHeader } from "../../src/components/routeLadder"
+
+const meta = {
+  component: RouteLadderHeader,
+  args: {
+    routeName: "39",
+    onClose: () => {},
```

Comment:
Could you replace this with `fn()` from `@storybook/test`?
https://storybook.js.org/docs/essentials/actions#via-storybooktest-fn-spy-function

Reply:
Agh ok! I hadn't started looking into the change, so you saved me before I had begun the wild goose chase